great_expectations/cli/upgrade_helpers/upgrade_helper_v13.py

```diff
@@ -21,7 +21,7 @@ def __init__(
         self,
         data_context: Optional[DataContext] = None,
         context_root_dir: Optional[str] = None,
-        update_version: Optional[bool] = False,
+        update_version: bool = False,
```

Comment:
@Shinnnyshinshin This seems like a proper change, yet a couple of tests started failing...  Not sure why.